swagger.yaml

```diff
@@ -2590,6 +2606,11 @@ definitions:
     additionalProperties:
       type: string
 
+  openshift-versions:
+    type: object
+    additionalProperties:
+      $ref: '#/definitions/openshift-version'
```

Comment:
Any compelling reason to do it like this, and not more well-defined as an array of objects with "version" and "release_image" properties?

Reply:
Because we have a hashmap and we always call a certain OCP version data by its version key.
Instead of having an array and wrapping it with `map[string]models.OpenShiftVersion`, swagger generates it for me as `models.OpenshiftVersions`